charts/k8s-infra/values.yaml

```diff
@@ -657,10 +657,10 @@ otelDeployment:
   # -- OtelDeployment deployment annotation.
   annotations: {}
   # -- OtelDeployment pod(s) annotation.
-  podAnnotations:
-    signoz.io/scrape: 'true'
-    signoz.io/port: '8888'
-    signoz.io/path: /metrics
+  podAnnotations: {}
+    # signoz.io/scrape: 'true'
+    # signoz.io/port: '8888'
```

Comment:
I understood the ClickHouse part. Can you help me understand this piece? If someone were to run k8s-infra independently will they be able to get their agent's internal metrics (without creating override.yaml)?

Reply:
for independent k8s-infra, it is still a noop having those pod annotation.

We previously discussed about enabling `external.signoz.io/scrape` but never did it since it involves prometheus receiver + otlp + clickhousemetrics write (with resource to telemetry disabled). It could cause the previously seen redundant metrics issue.